ScoutSuite/providers/azure/services/network.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+
+from ScoutSuite.providers.azure.configs.base import AzureBaseConfig
+
+
+class NetworkConfig(AzureBaseConfig):
+    targets = (
+        ('network_watchers', 'Network Watchers', 'list_all', {}, False),
+        ('network_security_groups', 'Network Security Group', 'list_all', {}, False),
+    )
+
+    def __init__(self, thread_config):
+        self.network_watchers = {}
+        self.network_watchers_count = 0
+
+        self.network_security_groups = {}
+        self.network_security_groups_count = 0
+
+        super(NetworkConfig, self).__init__(thread_config)
+
+    def parse_network_watchers(self, network_watcher, params):
+        network_watcher_dict = {}
+        network_watcher_dict['id'] = network_watcher.id
+        network_watcher_dict['name'] = network_watcher.name
+        network_watcher_dict['provisioning_state'] = network_watcher.provisioning_state == "Succeeded"
```

Comment:
I find this name a bit confusing, I would expect some kind of state, but it contains a boolean. I'd rename to something like `is_provisioning`, or put the state directly (since it has three possible values: creating, resolvingDNS or succeeded).